code/modules/mob/living/silicon/robot/laws.dm

```diff
@@ -4,6 +4,9 @@
 	show_laws()
 
 /mob/living/silicon/robot/show_laws()
+	if (istype(src, /mob/living/silicon/robot/shell))
```

Comment:
make /mob/living/silicon/robot/shell/show_laws() and have it return on the first line